src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitJobServlet.groovy

```diff
@@ -147,6 +147,28 @@ class GrabbitJobServlet extends SlingAllMethodsServlet {
         response.writer.write(new JsonBuilder(jobExecutionIds).toString())
     }
 
+    @Override
+    protected void doPost(SlingHttpServletRequest request, SlingHttpServletResponse response)  {
+        String jobExecutionId = request.getParameter("jobId") ?: ""
+        if(!jobExecutionId.isLong()) {
+            log.warn "Parameter ${jobExecutionId} 'jobId' is invalid"
+            response.status = SC_BAD_REQUEST
+            response.writer.write("Parameter 'jobId' is invalid")
+            return
+        }
+        try {
```

Comment:
@jbornemann : Lets say I have lots of jobs in grabbit repo that were run some time in past and after that instance was restarted, then if I say stop job on one of those jobs, it will give throw NoSuchJobExecutionException instead of JobExecutionNotRunningException... 
If we decide to remove those jobs in addition to stopping, then two exception blocks would be more useful...